test/k6/src/tests/instances.js

```diff
@@ -291,6 +297,50 @@ function TC11_HardDeleteInstance(data) {
   addErrorCount(success);
 }
 
+//TC12 - Get all instances for party looked up with a person number
+function TC12_GetInstances_ByPersonNumber(data) {
+  var instanceOwnerIdentifier = "Person:" + data.personNumber;
+  var res = instancesApi.getInstanceByInstanceOwnerIdentifier(data.userToken, instanceOwnerIdentifier);
+
+  var success = check(res, {
+    "TC12_GetInstances_ByPersonNumber: Get instance for party. Status is 200": (
+      r
+    ) => r.status === 200,
+  });
+  addErrorCount(success);
+
+  const firstInstance = JSON.parse(res.body).instances[0];
+  const instanceIdSplit = firstInstance["id"].split("/");
+
+  success = check(res, {
+    "TC12_GetInstances_ByPersonNumber: Get instance for party. InstanceId has expected format":
+      instanceIdSplit.length === 2,
+  });
+  addErrorCount(success);
+}
+
+//TC13 - Get all instances for party looked up with a person number
+function TC13_GetInstances_ByOrgNumber(data) {
+  var instanceOwnerIdentifier = "Organisation:" + data.orgNumber;
+  var res = instancesApi.getInstanceByInstanceOwnerIdentifier(data.userToken, instanceOwnerIdentifier);
+
+  var success = check(res, {
+    "TC13_GetInstances_ByOrgNumber: Get instance for party. Status is 200": (
+      r
+    ) => r.status === 200,
+  });
+  addErrorCount(success);
+
+  const firstInstance = JSON.parse(res.body).instances[0];
+  const instanceIdSplit = firstInstance["id"].split("/");
+
+  success = check(res, {
```

Comment:
A more interesting check here might be to check instanceOwner.organisationNumber to see if this matches the org number used as the instance owner identifier.